package.json

```diff
@@ -1,128 +1,128 @@
+		"generate-database-types": "dotenv -e ./.env/.env.local graphql-codegen -- --config codegen-hetarchief.yml"
+	},
+	"dependencies": {
+		"@nestjs/common": "8.2.3",
+		"@nestjs/config": "1.1.5",
+		"@nestjs/core": "8.2.3",
+		"@nestjs/platform-express": "8.2.3",
+		"@nestjs/schedule": "1.0.2",
+		"@nestjs/swagger": "5.2.0",
+		"@studiohyperdrive/pagination": "1.0.0",
+		"@viaa/avo2-types": "2.46.3",
+		"@meemoo/admin-core-api": "2.0.28",
+		"app-root-path": "3.0.0",
+		"aws-sdk": "2.1097.0",
+		"blend-promise-utils": "1.28.0",
+		"cache-manager": "3.6.0",
+		"class-transformer": "0.5.1",
+		"class-validator": "0.13.2",
+		"connect-redis": "6.0.0",
+		"date-fns": "2.28.0",
+		"date-fns-tz": "1.3.0",
+		"express-session": "1.17.2",
+		"fs-extra": "10.0.0",
+		"got": "11.8.3",
+		"graphql": "15.8.0",
+		"helmet": "5.0.2",
+		"i18next": "21.6.3",
+		"joi": "17.5.0",
+		"lodash": "4.17.21",
+		"node-cron": "3.0.0",
+		"query-string": "7.1.0",
+		"redis": "3.1.2",
+		"reflect-metadata": "0.1.13",
+		"rimraf": "3.0.2",
+		"rxjs": "7.4.0",
+		"saml2-js": "3.0.1",
+		"session-file-store": "1.5.0",
+		"swagger-ui-express": "4.3.0",
+		"uuid": "8.3.2",
+		"xml-js": "1.6.11"
+	},
+	"devDependencies": {
+		"@commitlint/cli": "15.0.0",
+		"@commitlint/config-conventional": "15.0.0",
+		"@graphql-codegen/cli": "2.6.2",
+		"@graphql-codegen/introspection": "2.1.1",
+		"@graphql-codegen/typed-document-node": "2.2.8",
+		"@graphql-codegen/typescript": "2.4.8",
+		"@graphql-codegen/typescript-operations": "2.3.5",
+		"@graphql-codegen/typescript-resolvers": "2.6.1",
+		"@graphql-typed-document-node/core": "3.1.1",
+		"@nestjs/cli": "8.1.5",
+		"@nestjs/schematics": "8.0.5",
+		"@nestjs/testing": "8.2.3",
+		"node-fetch": "2.6.1",
+		"@types/cache-manager": "3.4.3",
+		"@types/connect-redis": "0.0.18",
+		"@types/cron": "1.7.3",
+		"@types/express": "4.17.13",
+		"@types/express-session": "1.17.4",
+		"@types/fs-extra": "9.0.13",
+		"@types/helmet": "4.0.0",
+		"@types/jest": "27.0.3",
+		"@types/lodash": "4.14.178",
+		"@types/multer": "1.4.7",
+		"@types/node": "16.11.10",
+		"@types/redis": "2.8.32",
+		"@types/session-file-store": "1.2.2",
+		"@types/supertest": "2.0.11",
+		"@types/uuid": "8.3.4",
+		"@typescript-eslint/eslint-plugin": "5.4.0",
+		"@typescript-eslint/parser": "5.4.0",
+		"dotenv": "10.0.0",
+		"dotenv-cli": "5.1.0",
+		"eslint": "^8.29.0",
+		"eslint-config-prettier": "^8.5.0",
+		"eslint-import-resolver-typescript": "^3.5.2",
+		"eslint-plugin-import": "^2.26.0",
+		"eslint-plugin-prettier": "^4.2.1",
+		"glob": "7.2.0",
+		"husky": "7.0.4",
+		"jest": "27.3.1",
+		"lint-staged": "12.1.2",
+		"nock": "13.2.2",
+		"prettier": "2.5.0",
+		"redis-mock": "0.56.3",
+		"sort-object-keys": "1.1.3",
+		"source-map-support": "0.5.21",
+		"supertest": "6.1.6",
+		"ts-jest": "27.0.7",
+		"ts-loader": "9.2.6",
+		"ts-node": "10.4.0",
+		"tsconfig-paths": "3.12.0",
+		"typescript": "4.5.5"
+	}
```

Comment:
SHD guidelines prefer 2 spaces for package.json and package-lock.json files
So we might need to update the prettier config to add a special case for these files:

```
{
  "useTabs": true,
  "tabWidth": 4,
  "trailingComma": "es5",
  "singleQuote": true,
  "endOfLine": "auto",
  "printWidth": 100,
  "overrides": [
	{
	  "files": "{package,package-lock}.json",
	  "options": {
		"useTabs": false,
		"tabWidth": 2
	  }
	}
  ]
}
```

We want to have 2 spaces for package.json, because that is how the file is formatted when you do an npm install command. And otherwise we risk sometimes committing with tabs, and sometimes with spaces